Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -450,6 +459,16 @@ static String mutationObserverNotificationString()
         m_overflowControlsContainer->setAnchorPoint({ });
         m_rootLayer->addChild(*m_overflowControlsContainer);
     }
+
+#if ENABLE(UNIFIED_PDF_PAINT_SELECTIONS)
+    if (!m_selectionsLayer) {
+        // FIXME: Evaluate whether this should be a TiledBacking layer and hooked up to an async renderer.
```

Comment:
Yes it should be (it can get as large as the contents). No need to hook up the async renderer.

Reply:
Addressed in https://github.com/WebKit/WebKit/commit/3175c379f8966936bbf4d9833ad4b1c11e7a1ede